packages/server/src/config.ts

```diff
@@ -111,6 +111,31 @@ function baseUrlFromEnv(env: EnvVariables, appPort: number): string {
 	}
 }
 
+function ldapConfigFromEnv(env: EnvVariables): LdapConfig[] {
+	return [
+		{
+			enabled: env.LDAP_1_ENABLED,
+			userCreation: env.LDAP_1_CREATE_USER,
+			host: env.LDAP_1_SERVER,
```

Comment:
Is it possible to use the same term for both? Either "host" or "server" for both